renderer/components/Settings/messages.js

```diff
@@ -32,6 +32,8 @@ export default defineMessages({
   address_option_np2wkh: 'Segwit',
   blockExplorer_label: 'Block explorer',
   blockExplorer_description: 'Your preferred block explorer.',
+  fallbackAddress_label: 'Enable on-chain address fallback',
+  fallbackAddress_description: 'Add fallback on-chain address to LN invoices.',
```

Comment:
Can we reword this to `Add on-chain fallback address to LN invoices.`

"on-chain fallback address"
vs
"fallback on-chain address" 

We know it's a fallback address, the question is what type of fallback address. The type qualifier should come first.